lib/Cro/WebApp/Template/ASTBuilder.pm6

```diff
@@ -297,9 +335,12 @@ class Cro::WebApp::Template::ASTBuilder {
         return @squashed;
     }
 
-    sub escape($target) {
+    sub escape($/, $target, $pos) {
+        without &line-calculator {
+            &line-calculator = calculate-offset-to-lines($/.orig);
+        }
```

Comment:
Or just `//=`, which is probably a little cheaper. However, I wonder - since we'll always need it - if we can also always calculate it somewhere in advance.